constellation_forms/tests/models/testFormSubmission.py

```diff
@@ -0,0 +1,224 @@
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+from django.utils import timezone
+from ...models import Form
+from ...models import FormSubmission
+
+
+class FormSubmissionModelTest(TestCase):
+    def test_good_submission(self):
```

Comment:
This could be cleaner if you put a lot of the duplicate code into a single function.  Thoughts @the-maldridge ?